zipline/finance/cancel_policy.py

```diff
@@ -0,0 +1,70 @@
+import abc
+import logbook
+
+from six import with_metaclass, iteritems
+
+from zipline.gens.sim_engine import DAY_END
+from zipline.utils.serialization_utils import (
+    VERSION_LABEL
+)
+
+warning_logger = logbook.Logger('AlgoWarning')
+
+
+def log_warning(amount, sid, filled, policy_name):
+    warning_logger.warn(
```

Comment:
@richafrank one distinction is that `warnings` are easier to silence, but also only fire once by default.  I think logging here makes sense, since each trigger of this is a distinct event that shouldn't be amalgamated with other cancellation warnings.